packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@schema-data-loader/core",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
You should not bump version manually it's automatically handled by the GitHub action.

Reply:
Thanks Guido! Long time no see, hope everything is well :)